_data/devops-checklist.yml

```diff
@@ -0,0 +1,690 @@
+
+    - title: Deploy AMIs using Auto Scaling Groups
+      description: |
+        The best way to deploy an AMI is typically to run it as an
+        <a href="https://docs.aws.amazon.com/autoscaling/ec2/userguide/AutoScalingGroup.html" target="_blank">Auto Scaling Group</a>.
+        This will allow you to spin up multiple EC2 Instances that run your AMI, scale the number of Instances up and
+        down in response to load, and automatically replace failed Instances.
+
+    - title: Build Docker images
+      description: |
+        If want to run your apps as containers, you should package your apps as
+        <a href="http://docker.com/" target="_blank">Docker</a> images and push those images to Amazon's
+        <a href="https://aws.amazon.com/ecr/" target="_blank">Elastic Container Registry (ECR)</a>. We recommend
+        Docker for all stateless apps and for local development (along with
+        <a href="https://docs.docker.com/compose/" target="_blank">Docker Compose</a>).
+
+    - title: Deploy Docker images using ECS, EKS, or Fargate
+      description: |
+        You have several options for running Docker containers in AWS. One is to use the
+        <a href="https://aws.amazon.com/ecs/" target="_blank">Elastic Container Service (ECS)</a>, where you run a
+        cluster of EC2 Instances, and Amazon takes care of scheduling containers across them. Another is
+        <a href="https://aws.amazon.com/eks/" target="_blank">Elasic Kubernetes Service (EKS)</a>, which is a Amazon's
+        managed <a href="https://kubernetes.io/" target="_blank">Kubernetes</a> (note, EKS is still in preview mode
+        as of May, 2018). A third option is <a href="https://aws.amazon.com/fargate/" target="_blank">AWS Fargate</a>,
+        a service where AWS manages and scales the underlying Instances for you and you just hand it Docker containers
+        to run.
+
+    - title: Deploy serverless apps using Lambda and API Gateway
+      description: |
+        If want to build serverless apps, you should package them as
+        <a href="https://docs.aws.amazon.com/lambda/latest/dg/deployment-package-v2.html" target="_blank">deployment packages</a>
+        for <a href="https://aws.amazon.com/lambda/" target="_blank">AWS Lambda</a>. You can expose your Lambda
+        functions as HTTP endpoints using <a href="https://aws.amazon.com/api-gateway/" target="_blank">API Gateway</a>.
+
+    - title: Configure CPU, memory, and GC settings
+      description: |
+        Configure memory settings CPU settings (e.g., ensure your app uses all available CPU cores using tools such
+        as Node Cluster), memory settings (e.g., <code>-Xmx, -Xms</code> settings for the JVM), and GC settings (if
+        applicable) for your app. If you're deploying directly on EC2 Instances, these should be configured based on
+        the available CPU and memory on your Instance (see
```

Comment:
It seems you've purposely capitalized `Instances` when you refer to EC2 instances.  I'm not entirely sure it should be capitalized, especially when you just say something like "...on your Instance...". Seems like it should be `instance` there.

Reply:
Yea, we try to capitalize AWS Terminology: EC2 Instance, Security Group, etc. Not sure what the "proper" way is, but it's what we've been doing so far.

Reply:
Makes sense!